src/app/Pages/Dashboard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import Card from '../components/Card/Card';
+import useLocalStorage from '../hooks/useLocalStorage';
+
+type Diary = { destination: string; date: string; memories: string };
+
+export default function Dashboard(): JSX.Element {
+  const [diaries, setDiaries] = useLocalStorage<Diary[]>('diaries', []);
+
+  function deleteDiary(diary: {
+    destination: string;
+    date: string;
+    memories: string;
+  }) {
+    const newDiarylist = diaries.filter((singleDiary) => singleDiary !== diary);
+    setDiaries(newDiarylist);
+  }
+
+  return (
+    <Container>
+      <Link to="/addMemories">
+        <Button>AddMemories</Button>
+      </Link>
+      <Cardcontainer>
+        {diaries &&
+          diaries.map((diary, key) => (
+            <Card
+              diary={diary}
+              key={`${diary.destination}-${key}`}
+              deleteCard={() => deleteDiary(diary)}
```

Comment:
I suppose that by diary you mean a single entry. If so, it would be better to call it diaryEntry instead of simply diary, because it creates a bit of confusion like this. since a diary is essentially a collection of daily entries.